engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-system-purpose-modal.html

```diff
@@ -0,0 +1,77 @@
+<div data-extend-template="components/views/bst-modal.html">
+  <h4 data-block="modal-header" translate>Content Host System Purpose</h4>
+
+  <div data-block="modal-body">
+    <h4 translate>Assign System Purpose:</h4>
+
+    <div class="row">
+      <div class="col-sm-12">
+        <div bst-global-notification></div>
+      </div>
+    </div>
+
+    <form name="contentHostContentForm" class="form" ng-hide="content.workingMode">
+      <div>
+        <label translate>Service Level:</label>
+
+        <select type="select"
+                ng-options="item for item in defaultServiceLevels"
+                ng-model="selectedServiceLevels">
+        </select>
+      <br /><br />
+
+        <label translate>Usage Type:</label>
+
+        <select type="select"
+                readonly="denied('edit_hosts', host)"
+                ng-options="item for item in defaultUsages"
+                ng-model="selectedRoles">
+        </select>
+        <br /><br />
+
+        <label translate>Role:</label>
+
+        <select type="select"
+                readonly="denied('edit_hosts', host)"
+                ng-options="item for item in defaultRoles"
+                ng-model="selectedUsages">
+        </select>
+        <br /><br />
+
+        <label ng-hide="purposeAddonsList.length === 0" translate>Add ons:</label>
```

Comment:
```suggestion
        <label ng-hide="purposeAddonsList().length === 0" translate>Add ons:</label>
```
This is a function now, so remember to call the function :wink: 

If you want to be fancy, `0` in JavaScript is falsey, so instead of `purposeAddonsList().length === 0` you could just write `!purposeAddonsList().length`.  But some prefer the non-fancy version which is more explicit.  Whatever you're more drawn to is good with me.

Reply:
Updated to the fancy way :)